2023/07.md

```diff
@@ -110,6 +110,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |   | 1     | 60m     | Decimal: Open-ended discussion ([slides](https://docs.google.com/presentation/d/19MaO7On6knlweYZUei-d5VyqANzkKeR8QmC13KATvgs/)) | Jesse Alama |
     |   | 0     | 15m     | [DataView get/set Uint8C methods](https://github.com/ljharb/proposal-dataview-get-set-uint8c) for stage 1 or 2 or 3 ([spec](https://ljharb.github.io/proposal-dataview-get-set-uint8c/)) | Jordan Harband |
     |   | 0     | 25m     | Optional chaining in assignment LHS for stage ? ([motivation](https://gist.github.com/nicolo-ribaudo/d264e424b618e7deaeca1d6e4f16a7c0), [spec](https://github.com/nicolo-ribaudo/ecma262/pull/4), slides will come) | Nicolò Ribaudo |
+    |   | 0     | 20m     | [Source Phase Imports](https://github.com/tc39/proposal-source-phase-imports) for Stage 3 ([spec](https://tc39.es/proposal-source-phase-imports/), slides pending) | Guy Bedford & Luca Casonato |
```

Comment:
This proposal is not currently stage 0. It is stage 2. Please correct and also order as required by the [agenda topic rules](https://github.com/tc39/agendas/blob/main/2023/07.md#agenda-topic-rules).

Reply:
Also 20 minutes for *any* stage 3 advancement is ambitious. I would personally want a longer timebox.

Reply:
Thanks for the feedback, missed that one on the copy. I've bumped the timebox to 30m.

Reply:
still needs to be reordered

Reply:
👍 I've added the reordering and a 15 minute buffer on the time as well.